docs/generators/changelog/versions/2.11.md

```diff
@@ -25,6 +25,7 @@
 * Add type definitions for async methods from Meteor 2.9 by [GH ebroder] [PR #12507]
 * TypeScript skeleton fixes by [GH ebroder] [PR #12506]
 * Fix TypeScript type dependencies for mongo, webapp, and underscore by [GH ebroder] [PR #12505]
+* Improve specificity of types previously declared as "Object" by [GH ebroder] [PR #12520]
```

Comment:
Node update is missing here.

Reply:
Nice! Done. 
I will make recheck again when I start working on the official release for this one
(Wished that gh had a page showing wich prs were merged in branch)